tests/integration/ha_tests/test_upgrade.py

```diff
@@ -32,11 +32,17 @@
 @pytest.mark.abort_on_fail
 async def test_deploy_latest(ops_test: OpsTest) -> None:
     """Simple test to ensure that the PostgreSQL and application charms get deployed."""
-    await ops_test.model.deploy(
+    await ops_test.juju(
```

Comment:
This will be rolled back as soon as a revision is published to the `16/edge` channel.